packages/destination-actions/src/destinations/antavo/profile/index.ts

```diff
@@ -0,0 +1,99 @@
+import type { ActionDefinition } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from './generated-types'
+
+const action: ActionDefinition<Settings, Payload> = {
+  title: 'Profile updates',
+  description: 'Sync profile updates into Antavo',
+  defaultSubscription: 'type = "identify"',
+  fields: {
+    customer: {
+      label: 'Customer ID',
+      description: 'User ID, selected in Antavo as customer identifier',
+      type: 'string',
+      required: true
+    },
```

Comment:
Adding a default mapping so that userId is automatically mapped. 

```suggestion
    customer: {
      label: 'Customer ID',
      description: 'User ID, selected in Antavo as customer identifier',
      type: 'string',
      required: true,
      default: {
        '@path': '$.userId'
      }
    },
```

Reply:
hi @joe-ayoub-segment this is now added